src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -386,6 +391,11 @@
           "type": "string"
         },
 
+        "decoratedName": {
+          "description": "The fully qualified name for the logical location, in the compiler's native format.",
```

Comment:
Do what you think is best, then +1. I'm also glad to review your revision if you'd prefer before merging. 

---
In reply to: [63254879](https://github.com/Microsoft/sarif-sdk/pull/272#discussion_r63254879) [](ancestors = 63254879)